src/api/app/services/workflows/yaml_to_workflows_service.rb

```diff
@@ -33,7 +35,7 @@ def create_workflows
       end
     end
 
-    def parse_workflows_file(file_path)
+    def parse_workflows_content(workflows_file_content)
```

Comment:
You now use *4* different names for the same thing

- workflows_file_content
- workflow_configuration_yaml
- workflows_content
- (parsed_)workflows_yaml

Can we reduce this to 1 please?

Reply:
Yes, of course

Reply:
DoneDone